(dp1
S'output'
p2
S''
sS'layer'
p3
S'/home/dave/web2py/applications/init/controllers/authenticate.py'
p4
sS'code'
p5
S"from gluon.storage import Storage\nexec('from applications.%s.modules import gfc_auth' %request.application)\nreload(gfc_auth);\n\n#check if the current user is logged in\ndef index():\n\n    if not session.logged_in:\n        session.user = gfc_auth.GFCUser.authenticate(mixtree_conf, request)\n        if session.user:\n            #create an auth session entry so that we can use the\n            #SimpleAuth.isLoggedIn method and other decorators\n            #based on gluon.Auth\n            session.logged_in = True\n    \n    return dict()\n\n\nresponse._vars=response._caller(index)\n"
p6
sS'traceback'
p7
S'Traceback (most recent call last):\n  File "/home/dave/web2py/gluon/restricted.py", line 184, in restricted\n    exec ccode in environment\n  File "/home/dave/web2py/applications/init/controllers/authenticate.py", line 2, in <module>\n    exec(\'from applications.%s.modules import gfc_auth\' %request.application)\n  File "<string>", line 1, in <module>\n  File "/home/dave/web2py/applications/init/modules/gfc_auth.py", line 14, in <module>\n    class GFCUser:\n  File "/home/dave/web2py/applications/init/modules/gfc_auth.py", line 15, in GFCUser\n    self.name\nNameError: name \'self\' is not defined\n'
p8
s.